src/basic/conf-files.h

```diff
@@ -8,8 +8,9 @@ typedef enum ConfFilesFlags {
         CONF_FILES_REGULAR                  = 1 << 1,
         CONF_FILES_DIRECTORY                = 1 << 2,
         CONF_FILES_BASENAME                 = 1 << 3,
-        CONF_FILES_FILTER_MASKED_BY_SYMLINK = 1 << 4,
-        CONF_FILES_FILTER_MASKED_BY_EMPTY   = 1 << 5,
+        CONF_FILES_CHASE_BASENAME           = 1 << 4, /* If set, also resolve symlink files in config directories. */
```

Comment:
a flag like might make sense, but "basename" is a bit of a misnomer, no? maybe call this _FULL_CHASE? or _RESOLVE_PATH?